### DISCUSSION TITLE
Auto update feature for DAX Studio

### DISCUSSION THREAD

**Discussion by mahesh-gh** [DaxStudio/DaxStudio#596] [General]
Is there an auto update feature available for DAX application?

**Comment by dgosbell** [comment]
No there is no auto update feature. But you should get version update notifications once a day if you are running on an older release.

The problem with autoupdates is that they need to run **without** admin rights. And we have a number of integration points like the Power BI External Tools ribbon and the Excel addin which **require** admin rights in order to be registered.
